src/LongRunning/OperationsClient.php

```diff
@@ -49,11 +49,11 @@
 use Google\GAX\GrpcCredentialsHelper;
 use Google\GAX\PageStreamingDescriptor;
 use Google\GAX\ValidationException;
-use google\longrunning\CancelOperationRequest;
-use google\longrunning\DeleteOperationRequest;
-use google\longrunning\GetOperationRequest;
-use google\longrunning\ListOperationsRequest;
-use google\longrunning\OperationsGrpcClient;
+use Google\Longrunning\CancelOperationRequest;
+use Google\Longrunning\DeleteOperationRequest;
+use Google\Longrunning\GetOperationRequest;
+use Google\Longrunning\ListOperationsRequest;
+use Google\Longrunning\OperationsGrpcClient;
```

Comment:
Because these are protobuf-generated, we aren't able to fix this at the moment. Potentially we can have the PHP protoc implementation accept a namespace option, similar to C#s (https://github.com/googleapis/googleapis/blob/master/google/longrunning/operations.proto#L24)